build/dockerfiles/rhel.Dockerfile

```diff
@@ -13,11 +13,11 @@
 # Build registry
 # UPSTREAM: use RHEL7/RHSCL/httpd image so we're not required to authenticate with registry.redhat.io
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhscl/httpd-24-rhel7
-FROM registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-133.1608869526
+FROM registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-143.1619800879
```

Comment:
should we not switch instead to `ubi8/httpd-24` ?

Reply:
> # UPSTREAM: use RHEL7/RHSCL/httpd image so we're not required to authenticate with registry.redhat.io

Reply:
I believe this is one of those situations where the image that's available to the public is on one URL/registry and the equivalent image internally that we need for Brew is different. So we could maybe use ubi8/httpd upstream and rhel8/httpd downstream, or httpd-rhel7 up and rhel8/httpd down. 

Either way, needs to be different... and this works. Since we don't BUILD this rhel.dockerfile upstream in any tests, doesn't really matter what the hypothetical base image is. Or... do we have users that use `build.sh --rhel` here to use this dockerfile instead of the alpine one?

Reply:
> UPSTREAM: use RHEL7/RHSCL/httpd image so we're not required to authenticate with registry.redhat.io

I know that but ubi8/httpd-24 is not requiring authentication as well

Reply:
> Either way, needs to be different... and this works. Since we don't BUILD this rhel.dockerfile upstream in any tests, doesn't really matter what the hypothetical base image is. Or... do we have users that use build.sh --rhel here to use this dockerfile instead of the alpine one?


we do run happy path tests on both alpine and rhel images (and we check that image is building as well)

Reply:
If we can use the ubi8 image upstream without authentication then I'm +1 to that

Reply:
ubi8/httpd is new as of UBI 8.4, so if that works w/o auth, happy to switch to that